api/sales_dashboard/views.py

```diff
@@ -137,7 +137,15 @@ def organisation_info(request, organisation_id):
 
     # If self-hosted and running without an Influx DB data store, we don't want to/cant show usage
     if settings.INFLUXDB_TOKEN:
-        event_list, labels = get_event_list_for_organisation(organisation_id)
+        date_range = "30d"
```

Comment:
Couldn't all of this be simplified to: 

```python
date_range = request.GET.get("date_range", "30d")
context["date_range"] = date_range
```

The only thing that you lose with this approach I guess is that "date_range" is always set in the context, vs your approach which only sets it if it was included as a GET parameter. But, do we care?

Reply:
haha didnt even know you could do `request.GET.get("date_range", "30d")` so will just do that.

Reply:
OK - updated